herc/config.py

```diff
@@ -0,0 +1,53 @@
+import pyhocon
+import threading
+import traceback
+import sys
+import importlib
+
+_config = None
```

Comment:
Perhaps just have it load the config from an environment variable first, before looking in the files?  Then set the environment variable in tests?